src/seedu/addressbook/AddressBook.java

```diff
@@ -339,21 +343,21 @@ public static String executeCommand(String userInputString) {
         final String commandType = commandTypeAndParams[0];
         final String commandArgs = commandTypeAndParams[1];
         switch (commandType) {
-        case COMMAND_ADD_WORD:
+        case COMMAND_ADD_WORD :
```

Comment:
According to the [coding standard](https://oss-generic.github.io/process/codingstandards/coding-standards-java.html#white-space), under the White Space section, it says that colons should be surrounded by white space. The example given looks like part of a switch statement, which was the reason I made those changes. Was I mistaken?